src/@utils/provider.ts

```diff
@@ -56,9 +56,15 @@ export async function initializeProviderForCompute(
       accountId
     )
   } catch (error) {
-    const message = getErrorMessage(error.message)
-    LoggerInstance.error('[Initialize Provider] Error:', message)
-    toast.error(message)
+    const { message } = error
+    if (message.includes('algorithm_file_checksum_mismatch')) {
```

Comment:
suggestion: use a constant that can be quickly accessed and changed. Could be useful overall, maybe we can think about a "ocean_error_states" constant or something like that.